src/mame/juku/juku.cpp

```diff
@@ -132,17 +164,20 @@ void juku_state::bank_map(address_map &map)
 
 void juku_state::io_map(address_map &map)
 {
-	map(0x00, 0x01).rw(m_pic, FUNC(pic8259_device::read), FUNC(pic8259_device::write));
+	map(0x00, 0x03).rw(m_pic, FUNC(pic8259_device::read), FUNC(pic8259_device::write));
 	map(0x04, 0x07).rw(m_pio[0], FUNC(i8255_device::read), FUNC(i8255_device::write));
 	map(0x08, 0x0b).rw(m_sio[0], FUNC(i8251_device::read), FUNC(i8251_device::write));
 	map(0x0c, 0x0f).rw(m_pio[1], FUNC(i8255_device::read), FUNC(i8255_device::write));
 	map(0x10, 0x13).rw(m_pit[0], FUNC(pit8253_device::read), FUNC(pit8253_device::write));
 	map(0x14, 0x17).rw(m_pit[1], FUNC(pit8253_device::read), FUNC(pit8253_device::write));
 	map(0x18, 0x1b).rw(m_pit[2], FUNC(pit8253_device::read), FUNC(pit8253_device::write));
-	map(0x1c, 0x1f).rw(m_fdc, FUNC(fd1793_device::read), FUNC(fd1793_device::write));
+	map(0x1c, 0x1f).rw(m_fdc, FUNC(kr1818vg93_device::read), FUNC(kr1818vg93_device::write));
 	map(0x1c, 0x1c).w(FUNC(juku_state::fdc_cmd_w));
 	map(0x1f, 0x1f).rw(FUNC(juku_state::fdc_data_r), FUNC(juku_state::fdc_data_w));
-//  map(0x1c, 0x1d).rw(m_sio[1], FUNC(i8251_device::read), FUNC(i8251_device::write));
+	map(0x11,0x11).w(FUNC(juku_state::screen_cmd_a_w));
+	map(0x12,0x12).w(FUNC(juku_state::screen_data_a_w));
+	map(0x15,0x15).w(FUNC(juku_state::screen_cmd_b_w));
+	map(0x16,0x16).w(FUNC(juku_state::screen_data_b_w));
```

Comment:
This overwrites the PIT devices. At least a comment why this is done is needed here.